### PR TITLE
chore(HomeWorkSets): add the home work for sets

### PR DESCRIPTION
**Pull request by KurisuLim**
### Sets:
* Initiate a Set. Using For loop add integers from 1 to 10
* Remove all odd numbers from sets.
* Try to create a set with duplicate values

### CODE REVIEW

File: src/WeekFour/HomeworkSets.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        for(int i = 1; i <= number; i++ ){
+            loop_num.add(i);
+        }
+
+        for (int set : loop_num){
+            System.out.println("Counting: " + set);
+        }
+    }
+
+    public static void removeOddSet(int num){
```
Comment (vladimirKorzhukov) [inline src/WeekFour/HomeworkSets.java]: This is a good solution but you have to initialize and add numbers to the set and then remove all odd numbers from the set using "remove" method

File: src/WeekFour/HomeworkSets.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        removeOddSet(15);
+        System.out.println("\n#3");
+        duplicateValues();
+
+    }
+
+    public static void loopSet(int num){
+        int number = num;
+
+        Set<Integer> loop_num = new HashSet<>();
```
Comment (vladimirKorzhukov) [inline src/WeekFour/HomeworkSets.java]: Please use "camalCase" for variables.

File: src/WeekFour/HomeworkSets.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            loop_num.add(i);
+        }
+
+        for (int set : loop_num){
+            System.out.println("Counting: " + set);
+        }
+    }
+
+    public static void removeOddSet(int num){
+        Set<Integer> even_num = new HashSet<>();
```
Comment (vladimirKorzhukov) [inline src/WeekFour/HomeworkSets.java]: Please use "camalCase" for variables.

File: src/WeekFour/HomeworkSets.java (hunk 1 above)
Comment (KurisuLim) [reply src/WeekFour/HomeworkSets.java]: @vladimirKorzhukov 
Please recheck when you have the chance.

**Pull request merged**